src/test/client/nonblocking.cc

```diff
@@ -240,6 +250,33 @@ TEST_F(TestClient, LlreadvLlwritev) {
                        (const char*)iov_out_zero_bytes[1].iov_base,
                        iov_out_zero_bytes[1].iov_len));
 
+  // test async I/O with nullptr context
+  writefinish.reset(new C_SaferCond("test-nonblocking-writefinish-nullptr-context"));
+  readfinish.reset(new C_SaferCond("test-nonblocking-readfinish-nullptr-context"));
+
+  ssize_t nwritten_null_context = iov_out_null_context[0].iov_len
+                                  + iov_out_null_context[1].iov_len;
+
+  rc = client->ll_preadv_pwritev(fh, iov_out_null_context, 2, 0, true,
+                                 nullptr, nullptr);
+  ASSERT_EQ(0, rc);
+  rc = writefinish->wait();
```

Comment:
It seems this is incorrect, the wait() seems will be stuck infinitely.

Reply:
test cases in this file have used this, do you think using `wait_for()` will be better?

Reply:
Since the `context` is `nullptr`, so it may not need wait here.